pkg/ui/karma.conf.js

```diff
@@ -70,23 +67,23 @@ module.exports = function(config) {
     // preprocess matching files before serving them to the browser
     // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
     preprocessors: {
-      "ccl/src/**": ["webpack"],
-      "src/**": ["webpack"],
+      "src/polyfills.ts": ["webpack"],
+      "tests-loader.js": ["webpack", "sourcemap"],
     },
 
     // test results reporter to use
     // possible values: "dots", "progress"
     // available reporters: https://npmjs.org/browse/keyword/karma-reporter
-    reporters: ["mocha", "progress"],
+    reporters: ["mocha"],
 
     // Continuous Integration mode
     // if true, Karma captures browsers, runs the tests and exits
     singleRun: true,
 
     // https://github.com/airbnb/enzyme/blob/master/docs/guides/webpack.md
     webpack: {
-      devtool: "eval",
-      mode: "none",
+      devtool: "eval-cheap-source-map",
```

Comment:
Does this generate source maps? Was the goal to have source maps available for for test development? I'm guessing we could turn off source maps entirely.

Reply:
> Does this generate source maps? Was the goal to have source maps available for for test development? I'm guessing we could turn off source maps entirely.

Ideally it has to be an option to debug tests as well source files either within IDE or in browser (for this purposes Makefile has dedicated command `make ui-test-debug`.

Reply:
It’s out of scope for this PR, but I think I would like to see configs specifically for the CI environment. Then we could really strip out the overhead there and perhaps increase the memory for our local dev environments for debugging.